eng/build.yml

```diff
@@ -135,23 +135,13 @@ jobs:
         displayName: Build
         env:
           BuildConfig: $(_BuildConfig)
-      - script: eng/runTestsCannotRunOnHelix.sh
```

Comment:
Can this script be deleted now if it's no longer going to be used?

Reply:
It's still used in the new run helix and non helix in parallel script. At some point we'll need to get non-helix tests into helix but this is just parallelizing that through powershell for now.